admin/controller/payment/mollie/base.php

```diff
@@ -478,26 +478,57 @@ private function validate ($store = 0)
 	protected function checkModuleStatus ()
 	{
 		$need_files = array();
-		$mod_files  = array(
-			DIR_APPLICATION . "controller/payment/mollie/base.php",
-			DIR_APPLICATION . "language/english/payment/mollie.php",
-			DIR_TEMPLATE . "payment/mollie.tpl",
-			DIR_TEMPLATE . "payment/mollie_2.tpl",
-			DIR_CATALOG . "controller/payment/mollie-api-client/",
-			DIR_CATALOG . "controller/payment/mollie/base.php",
-			DIR_CATALOG . "language/english/payment/mollie.php",
-			DIR_CATALOG . "model/payment/mollie/base.php",
-			DIR_CATALOG . "view/theme/default/template/payment/mollie_checkout_form.tpl",
-			DIR_CATALOG . "view/theme/default/template/payment/mollie_return.tpl",
-			DIR_CATALOG . "view/theme/default/template/payment/mollie_return_2.tpl",
-		);
+		
+			if ($this->isOpencart22())
+			{
+			$mod_files  = array(
+				DIR_APPLICATION . "controller/payment/mollie/base.php",
+				DIR_APPLICATION . "language/en-gb/payment/mollie.php",
```

Comment:
Hello Maikel,

Thank you for your contribution to this module. Is it true that only the language follows a different structure in 2.2?